.env.local

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_SUPABASE_URL=https://rhqgxzkfhqfymaisznwd.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJocWd4emtmaHFmeW1haXN6bndkIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTY1ODE2MjgsImV4cCI6MjA3MjE1NzYyOH0.Uznhn15In_cZ9Z11vNhPLSjVyQEcuRY0GrtgFRm0UGc
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)